python/torch_mlir/dynamo.py

```diff
@@ -62,9 +63,12 @@ def _get_decomposition_table():
         aten.native_group_norm_backward,
         aten.sigmoid_backward,
         aten._native_batch_norm_legit,
-        aten._native_batch_norm_legit_no_training,
         aten.squeeze,
-    ])
+    ]
+    # TODO: enable test once 2.1.0 is stable
+    if version.parse(torch.__version__) > version.parse("2.0.1+cpu"):
```

Comment:
Nit: I think we can avoid the +CPU here.

Reply:
I tried it and it behaves differently from what one would expect. For example 
```
>>> version.parse("2.0.1") >= version.parse("2.0.1+cpu")
False
```
I would leave it there for now, if you are ok with it

Reply:
Sounds good.